views/partials/header.ejs

```diff
@@ -0,0 +1,10 @@
+<h1>The Jobs EJS Application</h1>
+<% if (errors) {
+  errors.forEach((err) => { %>
+    <div>Error: <%= err %></div>
+<% }) } %>
+<% if (info) {
+  info.forEach((msg) => { %>
+    <div>Info: <%= msg %></div>
+<% }) } %>
+<hr />
```

Comment:
Nitpick:
I'd recommend wrapping the `hr` tag in a conditional so that it doesn't render until there's a message:
```ejs
<% if ((errors && errors.length) || (info && info.length)) { %>
    <hr />
<% } %>
```